go.mod

```diff
@@ -105,6 +104,6 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
-go 1.13
+go 1.16
```

Comment:
@vito don't forget to update the Go version here as well:

https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md#development-dependencies

Reply:
good call!